static/js/image_upload_widget.js

```diff
@@ -12,6 +15,14 @@ function image_upload_start(spinner, upload_text, delete_button) {
     delete_button.hide();
 }
 
+function is_image_format(file) {
```

Comment:
Its a small function, but we can easily do something like:

`exports.is_image_format = (file) => { };`

in any of these two files (preferably image_upload_widget because that's where we want all its final consumers after the refactor to be).